common/main/as.xml

```diff
@@ -924,12 +924,16 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<type key="em" type="default" scope="core">ডিফ’ল্ট</type>
 			<type key="em" type="emoji" scope="core">ইম’জী</type>
 			<type key="em" type="text" scope="core">পাঠ্য</type>
+			<type key="hc" type="c12">১২ ঘণ্টীয়া প্ৰণালী</type>
+			<type key="hc" type="c12" scope="core">১২</type>
+			<type key="hc" type="c24">২৪ ঘণ্টীয়া প্ৰণালী</type>
+			<type key="hc" type="c24" scope="core">২৪</type>
 			<type key="hc" type="h11">১২ ঘণ্টীয়া প্ৰণালী (০–১১)</type>
 			<type key="hc" type="h11" scope="core">১২ (০–১১) ১২ (০–১১)</type>
 			<type key="hc" type="h12">১২ ঘণ্টীয়া প্ৰণালী (১–১২)</type>
 			<type key="hc" type="h12" scope="core">১২ (১–১২) ১২ (১–১২)</type>
 			<type key="hc" type="h23">২৪ ঘণ্টীয়া প্ৰণালী (০–২৩)</type>
-			<type key="hc" type="h23" scope="core">24 (0–23) Hour Cycle (12 vs 24) others… ২৪ (০–২৩) …অন্য ঘণ্টীয়া প্ৰণালী(১২ বনাম ২৪): ২৪ (০–২৩) ৩ (০–২৩)</type>
+			<type key="hc" type="h23" scope="core">২৪ (০–২৩) ২৪ (০–২৩)</type>
```

Comment:
```suggestion
			<type key="hc" type="h23" scope="core">২৪ (০–২৩)</type>
```

?

Reply:
I had filed a separate ticket to fix several instances like this: https://unicode-org.atlassian.net/browse/CLDR-19134